app/Http/Controllers/FrontendStatusController.php

```diff
@@ -226,7 +226,20 @@ public function usageboard(Request $request) {
         ]);
     }
 
+    /**
+     * @param $status
+     * @return mixed
+     * @todo move to Status Model and return StopOver instead of String
+     */
     public static function nextStation(&$status) {
```

Comment:
Can we just move this to JavaScript already? 😫

Reply:
Do it if you want. I'm out of that for now.